### PR TITLE
Final fixes

### PR DESCRIPTION
**Pull request by descorp**
# Open PR

## Changes

* update `example/android/app/build.gradle` to latest React-Native v0.71+ template
* fix "deprecated namespace in Manifest" warning
* update default example API version to v70 to enable telemetry
* fix placement of `checkoutAttemptId` in payment details payload

### CODE REVIEW

File: docs/Configuration.md [hunk 1]
```diff
@@ -1,48 +1,58 @@
...
+- **countryCode** - The shopper's country code in ISO 3166-1 alpha-2 format. Example: `"NL"` or `"US"`.
+- **shopperLocale** - 🚧 Work in progress. In current version this property only localises payment methods names. Default OS's locale is used for localisation.
+- **returnUrl** - For iOS, this is the URL to your app, where the shopper should return, after a redirection. Maximum of 1024 characters. For more information on setting a custom URL scheme for your app, read the [Apple Developer documentation](https://developer.apple.com/documentation/uikit/inter-process_communication/allowing_apps_and_websites_to_link_to_your_content/defining_a_custom_url_scheme_for_your_app).
+  For Android, this value is automatically overridden by `AdyenCheckout`.
+
+> ⚠️ To show the amount on the **Pay** button both _amount_ and _countryCode_ must be set.
+
+## Analytics
+
+- **enabled** - Enable/Disable all analytics. Defaults to `true`. ⚠️ This feature only availalbe fron v68 of Adyen API
```
Comment (goergisn) [inline docs/Configuration.md]:
```suggestion
- **enabled** - Enable/Disable all analytics. Defaults to `true`. ⚠️ This feature only available from v68 of Adyen API
```

File: docs/Configuration.md [hunk 2]
```diff
@@ -1,48 +1,58 @@
...
+- **payload** - Data to be sent along with the event data
 
 ## React Native SDK provides following configurations for components:
 
 ### DropIn component
-* **showPreselectedStoredPaymentMethod** - Determines whether to enable preselected stored payment method view step. Defaults to `true`.
-* **skipListWhenSinglePaymentMethod** - Determines whether to enable skipping payment list step when there is only one non-instant payment method. Defaults to `false`.
+
+- **showPreselectedStoredPaymentMethod** - Determines whether to enable preselected stored payment method view step. Defaults to `true`.
```
Comment (goergisn) [inline docs/Configuration.md]:
```suggestion
- **showPreselectedStoredPaymentMethod** - Determines whether to enable the preselected stored payment method view step. Defaults to `true`.
```

File: docs/Configuration.md [hunk 3]
```diff
@@ -1,48 +1,58 @@
...
-* **holderNameRequired** - Indicates if the field for entering the holder name should be displayed in the form. Defaults to false.
-* **hideCvcStoredCard** - Indicates whether to show the security code field on a stored card payment. Defaults to false.
-* **hideCvc** - Indicates whether to show the security code field at all. Defaults to false.
-* **addressVisibility** - Indicates the display mode of the billing address form. Options: `"none"`, `"postal"`, `"full"`. Defaults to `"none"`.
-* **kcpVisibility** - Indicates whether to show the security fields for South Korea issued cards. Options: `"show"` or `"hide"`. Defaults to `"hide"`.
-* **socialSecurity** - Indicates the visibility mode for the social security number field (CPF/CNPJ) for Brazilian cards. Options: "show" or `"hide"`. Defaults to `"hide"`.
-* **supported** - The list of allowed card types. By default uses list of `brands` from payment method. Fallbacks to list of all known cards.
+
+- **showStorePaymentField** - Indicates if the field for storing the card payment method should be displayed in the form. Defaults to true.
```
Comment (goergisn) [inline docs/Configuration.md]:
```suggestion
- **showStorePaymentField** - Indicates if the field for storing the card payment method should be displayed in the form. Defaults to `true`.
```

File: docs/Configuration.md [hunk 4]
```diff
@@ -1,48 +1,58 @@
...
+- **holderNameRequired** - Indicates if the field for entering the holder name should be displayed in the form. Defaults to false.
```
Comment (goergisn) [inline docs/Configuration.md]:
```suggestion
- **holderNameRequired** - Indicates if the field for entering the holder name should be displayed in the form. Defaults to `false`.
```

File: docs/Configuration.md [hunk 5]
```diff
@@ -1,48 +1,58 @@
...
+- **kcpVisibility** - Indicates whether to show the security fields for South Korea issued cards. Options: `"show"` or `"hide"`. Defaults to `"hide"`.
+- **socialSecurity** - Indicates the visibility mode for the social security number field (CPF/CNPJ) for Brazilian cards. Options: "show" or `"hide"`. Defaults to `"hide"`.
+- **supported** - The list of allowed card types. By default uses list of `brands` from payment method. Fallbacks to list of all known cards.
 
 ### ApplePay component
-* **merchantID** - The merchant identifier for apple pay.
-* **merchantName** - The merchant name. Used for generation of collection of `PKPaymentSummaryItem`.
-* **allowOnboarding** - The flag to toggle onboarding. If `true`, allow the shopper to add cards to Apple Pay if non exists yet. If `false`, Apple Pay is disabled when the shopper doesn’t have supported cards on Apple Pay wallet. Default is `false`.
+
+- **merchantID** - The merchant identifier for apple pay.
```
Comment (goergisn) [inline docs/Configuration.md]:
```suggestion
- **merchantID** - The merchant identifier for Apple Pay.
```

File: example/android/app/build.gradle [hunk 6]
```diff
@@ -1,135 +1,99 @@
...
+ * Set this to true to create four separate APKs instead of one,
+ * one for each native architecture. This is useful if you don't
+ * use App Bundles (https://developer.android.com/guide/app-bundle/)
+ * and want to have separate APKs to upload to the Play Store.
  */
 def enableSeparateBuildPerCPUArchitecture = false
 
 /**
- * Run Proguard to shrink the Java bytecode in release builds.
+ * Set this to true to Run Proguard on Release builds to minify the Java bytecode.
```
Comment (goergisn) [inline example/android/app/build.gradle:64]:
```suggestion
 * Set this to true to run Proguard on Release builds to minify the Java bytecode.
```

**Review by goergisn — COMMENTED** [review]
Looks good!

**Review by OscarSpruit — DISMISSED** [review]

**Review by ribeiroguilherme — DISMISSED** [review]

File: example/android/app/build.gradle (hunk 6 above)
Comment (descorp) [reply example/android/app/build.gradle:64]: This is React Native template - it is on Facebook to fix this 😆

File: docs/Configuration.md (hunk 5 above)
Comment (descorp) [reply docs/Configuration.md]: Fixed in [7567c07](https://github.com/Adyen/adyen-react-native/pull/232/commits/7567c071a9dbab954f3c478a2550fcc483717ef4)

**Review by goergisn — APPROVED** [review]

**Pull request merged**